Update Jira connector Example page (#448)
wso2/docs-mi: en/docs/reference/connectors/jira-connector/jira-connector-example.md

> # Jira Connector example

Reviewer: ```suggestion
# Jira Connector Example
```

Reviewer: Since this is the page topic, let's keep it in this way.

If you do not want to configure this yourself, you can simply [get the project](#get-the-project) and run it.

> ## Setup the Integration project

Reviewer: ```suggestion
## Set up the integration project
```

Follow the steps in the [create integration project]({{base_path}}/develop/create-integration-project/) guide to set up the Integration Project.

> ## Creating the Integration logic

Reviewer: ```suggestion
## Create the integration logic
```

1. Click `+` on the Extension panel APIs to create the REST API.

2. Provide the API name as `jiraAPI` and the API context as `/jira`. You can go to the source view of the XML configuration file of the API and copy the following configuration.
   ![Adding a Rest API]({{base_path}}/assets/img/integrate/connectors/jira-conn-1.png "Adding a Rest API")

> 3. Click the `/resource` default endpoint to open the **Resource View**. Then click the `+` arrow below the Start node to open the side panel. Select **Connectors** and search for the **jira** connector. Click the Jira connector to open the operation panel, then click **init**. This will download the connector.

Reviewer: ```suggestion
3. Click the `/resource` default endpoint to open the **Resource View**. Then click the `+` arrow below the Start node to open the side panel. Select **Connectors** and search for the **Jira** connector. Click the Jira connector to open the operation panel, then click **init**. This will download the connector.
```

   ![Adding a Jira Connector]({{base_path}}/assets/img/integrate/connectors/jira-conn-2.png "Adding a Jira Connector")

> 4. You can go to the source view (click the `</>` icon on the top right corner) of the XML configuration file of the API and copy the following configuration. Update values username,password and site url. For password use api key.

Reviewer: ```suggestion
4. You can go to the source view (click the `</>` icon on the top right corner) of the XML configuration file of the API and copy the following configuration. Modify username, password and site url. For password you can use `api key`.
```

Follow these steps to deploy the exported CApp in the integration runtime. 

**Deploying on Micro Integrator**
To deploy and run the project, refer to the [Build and Run]({{base_path}}/develop/deploy-artifacts/#build-and-run) guide.

> You can further refer the application deployed through the CLI tool. See the instructions on [managing integrations from the CLI]({{base_path}}/observe-and-manage/managing-integrations-with-apictl).

Reviewer: ```suggestion
You can further refer the application deployed through the CLI tool. See the instructions on [managing integrations from the CLI]({{base_path}}/observe-and-manage/managing-integrations-with-mi-cli).
```
